### DISCUSSION TITLE
MPh on WSL

### DISCUSSION THREAD

**Discussion by lrvieira** [MPh-py/MPh#146] [General]
My Comsol is installed on Windows, but for almost everything related to scripting I prefer to use Ubuntu on WSL2 (many libraries are a pain to install on Windows).
This might be a dumb question, but I'd like to know if it's possible to use MPh on Ubutuntu's side in my configuration, or if i would have to install Comsol on Ubuntu too.

**Comment by john-hen** [comment]
Pretty sure there's no other way than installing Comsol inside the Ubuntu VM as well. I doubt you can even start the Comsol installation on Windows from inside the Ubuntu shell. At least I can't, leads to all kind of errors. Plus, when MPh finds itself on a Linux system, it will start Comsol server processes with `comsol mphserver`, whereas on Windows it runs `comsolmphserver.exe`. So this alone would clash.
